### PR TITLE
Feat/add default processor

### PR DESCRIPTION
**Pull request by Yozhef**

### CODE REVIEW

File: src/Processor/ExceptionProcessor.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     * @param array<string, mixed> $record
+     *
+     * @return array[]
+     */
+    public function __invoke(array $record): array
+    {
+        if (
+            array_key_exists('context', $record) === false
+            || !array_key_exists('exception', $record['context'])
+            || !($record['context']['exception'] instanceof Throwable)
```
Comment (Yonchi) [inline src/Processor/ExceptionProcessor.php:21]: Why not 
```suggestion
            !array_key_exists('context', $record)
            || !array_key_exists('exception', $record['context'])
            || !($record['context']['exception'] instanceof Throwable)
```
OR
```suggestion
            array_key_exists('context', $record) === false
            || array_key_exists('exception', $record['context']) === false
            || !($record['context']['exception'] instanceof Throwable)
```

File: tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php [hunk 2]
```diff
@@ -22,21 +22,21 @@
...
 {
-    public function testNoExceptionOccurred()
+    public function testNoExceptionOccurred(): void
     {
         $message = new TestMessage('test');
         $envelope = new Envelope($message);
 
         $loggingStrategy = $this->createMock(LoggingStrategyInterface::class);
-        $loggingStrategy->expects($this->never())
+        $loggingStrategy->expects(self::never())
```
Comment (Yonchi) [inline tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php:31]: Why static methods ?

File: tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php [hunk 3]
```diff
@@ -109,21 +109,21 @@ public function testExceptionNotLogged()
         try {
             $middleware->handle($envelope, $this->getStackMock($handlerFailedException));
         } catch (Throwable $exception) {
-            $this->assertSame($handlerFailedException, $exception);
+            self::assertSame($handlerFailedException, $exception);
```
Comment (Yonchi) [inline tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php:112]: Why static methods ?

File: tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php (hunk 2 above)
Comment (Yozhef) [reply tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php:31]: We read this [post](https://discourse.laminas.dev/t/this-assert-vs-self-assert/448)  -> discussed in a team and then decided to use `self ::`

**Review by Yonchi — APPROVED** [review]

**Pull request merged**